@here/harp-mapview/lib/Utils.ts

```diff
@@ -486,6 +534,44 @@ export namespace MapViewUtils {
         );
     }
 
+    /**
+     * Extracts current camera tilt angle in radians.
+     *
+     * @param mapView The [[MapView]] in use.
+     */
+    export function extractCameraTilt(mapView: MapView): number {
+        const { camera, projection } = mapView;
```

Comment:
Why passing whole mapview?

Reply:
I don't like it too... but look at whole Utils.ts, almost everything there takes MapView, so I kindly adapted to _standard_ :). I changed mine stuff, but please don't ask me to fix all those methods :).